carsus/io/tests/test_gfall.py

```diff
@@ -13,11 +13,21 @@ def gfall_rdr(gfall_fname):
     return GFALLReader(fname=gfall_fname)
 
 
+@pytest.fixture()
+def gfall_rdr_http(gfall_http):
+    return GFALLReader(fname=gfall_http)
+
+
 @pytest.fixture()
 def gfall_raw(gfall_rdr):
     return gfall_rdr.gfall_raw
 
 
+@pytest.fixture()
+def gfall_raw_http(gfall_rdr_http):
+    return gfall_rdr_http.gfall_raw
```

Comment:
I'm not sure about this test because needs an internet connection and all the other Carsus tests seem to run offline.